test/extended/prometheus/prometheus.go

```diff
@@ -749,6 +749,10 @@ var _ = g.Describe("[sig-instrumentation] Prometheus [apigroup:image.openshift.i
 			if isManagedService {
 				allowedAlertNames = append(allowedAlertNames, "KubeDaemonSetMisScheduled")
 			}
+			// https://issues.redhat.com/browse/OCPBUGS-48340
+			if SkipOperatorHubMetricsCheck(oc) {
```

Comment:
I would suggest changing the name of the function to something like: `shouldSkipOperatorHubMetricsCheck`
Adding "should" or "is" makes it clear its a boolean check (as opposed to a command). I would also suggest not exporting it as it's an internal helper function.

```suggestion
			if shouldSkipOperatorHubMetricsCheck(oc) {
```

Reply:
Got it, thanks!

Reply:
This could be used for something more generic... You could even call it: `isDisableAllDefaultSources` or similar 🤷

Reply:
FYI, this is a backport from `main` where this has already merged. I don't think I'd put a nit name change in a backport PR. Otherwise future PR backports in this area will have conflicts.